### ISSUE TITLE
[fixed...see PR] Installation stalls on running npm run dev

### ISSUE THREAD

**Issue by connecteev** [fritsvt/laravel-nuxt-authentication#6]
I am seeing a new error on installation:

After going through the remaining steps, this is what the console shows (and it hangs) on running ```npm run dev```

```
ℹ Preparing project for development                                                                                                                                                               23:57:17
ℹ Initial build may take a while                                                                                                                                                                  23:57:17
✔ Builder initialized                                                                                                                                                                             23:57:17
✔ Nuxt files generated                                                                                                                                                                            23:57:17

● Client █████████████████████████ building (68%) 485/496 modules 11 active
 node_modules/bootstrap-vue/esm/utils/identity.js

✖ Server
  Compiled with some errors in 7.89s


✖ Client
  Compiled with some errors in 9.76s

✖ Server
  Compiled with some errors in 7.89s


 ERROR  Failed to compile with 1 errors                                                                                                                                           friendly-errors 23:57:30

This relative module was not found:                                                                                                                                               friendly-errors 23:57:30
                                                                                                                                                                                  friendly-errors 23:57:30
* ./social-callback.vue?vue&type=style&index=0&id=7446b537&lang=scss&scoped=true& in ./pages/auth/social-callback.vue                                                             friendly-errors 23:57:30
ℹ Waiting for file changes                                                                                                                                                                        23:57:30
ℹ Memory usage: 207 MB (RSS: 285 MB)                                                                                                                                                              23:57:30
^C
```

**Comment by connecteev** [comment]
Found the problem. package.json is missing node-sass and sass-loader:

```
diff --git a/client/package.json b/client/package.json
index 4afbfc0..e38e03b 100644
--- a/client/package.json
+++ b/client/package.json
@@ -16,7 +16,9 @@
     "bootstrap": "^4.1.3",
     "bootstrap-vue": "^2.0.0-beta",
     "cross-env": "^5.2.0",
-    "nuxt": "^2.0.0"
+    "npm-sass": "^2.3.0",
+    "nuxt": "^2.0.0",
+    "sass-loader": "^7.1.0"
   },
   "devDependencies": {

```

^^ this works, it should be simple to add, but let me know if you want a PR instead.

**Comment by connecteev** [comment]
See the PR I just submitted that fixes this problem, along with an updated readme with clear(er) instructions.

**Comment by fritsvt** [comment]
Merged in #7
